server/src/controller/auth.controller.ts

```diff
@@ -1,18 +1,22 @@
 import { Response, Request, NextFunction } from "express" 
-import db from '../db'
 import jwt from 'jsonwebtoken'
 import bcrypt from 'bcrypt'
+import { Admin } from '../models/Models'
 
 
 export const Auth = async (req: Request, res: Response) => {
 
     const { adminLogin, adminPassword } = req.body 
     
-    const credentials = await db.query('SELECT * FROM admin WHERE email = $1', [adminLogin])
+    const credentials = await Admin.findOne({
+        where: {
+            email: adminLogin
+        }
+    })
     
-    if(credentials.rows.length) {
+    if(credentials) {
 
-        const hashPass = credentials.rows[0].password
+        const hashPass = credentials.password
```

Comment:
Why do you need this value? I'd prefer two pick one of these:
1. const {password: hasPass} = credentials;
2. bcrypt.compare(adminPassword, creds.password)